contributing-guides/style-guide.md

```diff
@@ -481,6 +477,19 @@ When documenting optional placeholders like paths or file extensions, it is sugg
 
 - Use `{{path/to/source.ext}}` instead of `{{path/to/source.tar[.gz|.bz2|.xz]}}`.
 
+### Keypress syntax
+
+To mark keypresses for TUI or UI programs, use angle brackets `<` and `>`.
+
+- Single characters example: `<a>`.
+- Special keys are to be written with pascal case: `<Ctrl>`, `<Super>`, `<Alt>`, `<Shift>`, `<Cmd>`, `<Option>`, `<Windows>`, `<Enter>`, `<Home>`, `<Space>`, `<Esc>`, `<ArrowUp>`, `<ArrowLeft>`, `<PageUp>`, `<F5>`, `<F12>`, ...
+- Special keys can be translated if they have culturally relevant translations.
+- When a program takes in uppercase character literals mark them as `<A>` instead of marking it with shift. Otherwise always mark characters in lowercase.
+- Mark simultaneous keypresses inside the same angle brackets separated by a single space:  `<Ctrl c>`, `<Alt F4>`, `<Ctrl Shift k>`, `<Super Shift PrtSc>`.
+- Consecutive keypresses need to be contained in their own angle brackets with no space in between: `<:><w><q><Enter>`, `<Ctrl k><Ctrl s>`.
+- Keypresses in the same context or when they form words can be grouped together in the same angle brackets: `<:><help><Enter>`.
+- Do not place options with multiple choices inside angle brackets: Use `{{<Ctrl Shift PageUp>|<Ctrl Shift PageDown>}}` instead of `<Ctrl Shift {{PageUp|PageDown}}>`.
```

Comment:
Also consider if it's a good idea to put placeholders inside keypress brackets at all 
![image](https://github.com/user-attachments/assets/fdf293d5-8426-4c7b-b0f1-2d25e6042584)